### DISCUSSION TITLE
Can i work on a website for this amazing initiative?

### DISCUSSION THREAD

**Discussion by Neelanchal17** [carbon-language/carbon-lang#4443] [Getting started]
Hi, I'm Neelanchal, I have been following the incredible work you are doing with Carbon Language, I would like to contribute to this language, but I'm a website developer, As a website developer, I believe I can create a functional and visually appealing website for the Carbon Language project. I would like to seek the approval of the maintainers of this repository for making such a contribution. 
Thank you

**Comment by jonmeow** [comment]
Thanks for the interest!

Since that Carbon is still [under development](https://github.com/carbon-language/carbon-lang/?tab=readme-ov-file#project-status), we haven't spent too much time on the website, because we don't want to imply it's more ready than it is. If you'd like to work on the website, please bear that in mind.

Note our current website is available at https://docs.carbon-lang.dev/, and the supporting infrastructure can be found in the `website` directory. We're using GitHub Pages for serving and Jekyll for generation, in order to keep it easy to maintain while we focus on making Carbon ready-for-use.

If you want to discuss specific changes before trying to make them, #infra on [the Carbon Discord](https://discord.gg/ZjVdShJDAs) would be a good place to start.

**Reply by Neelanchal17** [reply]
Thank you for your response. I now understand that creating a full-fledged website may not be the best approach at the moment. However, I would be happy to work on it if a website for Carbon language is needed in the future. Thanks again!
